bigtable-hbase-replication/src/test/java/com/google/cloud/bigtable/hbase/replication/HbaseToCloudBigtableReplicationEndpointTest.java

```diff
@@ -204,23 +200,49 @@ public void testDelete() throws IOException, InterruptedException {
 
 
     int i =1; // Row 0 has been deleted.
-    for(Result result: cbtTable.getScanner(new Scan())){
+    assertResultRowsInCBT(cbtTable, 5, 1);
+  }
+
+  //TODO create a multi column family
+  @Test
+  public void testMultiTableReplication() throws IOException, InterruptedException {
+    Table table1 = hbaseTestingUtil.getConnection().getTable(TableName.valueOf(TABLE_NAME));
+    Table table2 = hbaseTestingUtil.getConnection().getTable(TableName.valueOf(TABLE_NAME_2));
+    for (int i = 0; i < 10; i++) {
+      String rowKey = ROW_KEY_PREFIX + i;
+      Put put = new Put(rowKey.getBytes());
+      byte[] val = (VALUE_PREFIX + i).getBytes();
+      put.addColumn(CF1, COL_QUALIFIER, 0, val);
+      table1.put(put);
+      table2.put(put);
+    }
+
+    // Wait for replication to catch up
+    // TODO Find a better alternative than sleeping? Maybe disable replication or turnoff mini cluster
+    Thread.sleep(2000);
+    //   Waiter.waitFor(CONF, 60000, () -> TestEndpoint.getEntries().size() >= cellNum);
+
+    Table cbtTable = cbtConnection.getTable(TableName.valueOf(TABLE_NAME));
+    Table cbtTable2 = cbtConnection.getTable(TableName.valueOf(TABLE_NAME_2));
+    assertResultRowsInCBT(cbtTable, 10, 0);
+    assertResultRowsInCBT(cbtTable, 10, 0);
+  }
+
+  private void assertResultRowsInCBT(Table table, int rowRange, int rowStart) throws IOException {
+    for(Result result: table.getScanner(new Scan())){
       LOG.error("Processing result: " + Bytes.toStringBinary(result.getRow()) + " cell: " + result.listCells().get(0).toString());
       List<Cell> returnedCells = result.listCells();
-      Assert.assertEquals(0, Bytes.compareTo(result.getRow(), (ROW_KEY_PREFIX + i).getBytes()));
+      Assert.assertEquals(0, Bytes.compareTo(result.getRow(), (ROW_KEY_PREFIX + rowStart).getBytes()));
       // Only 1 cell/row added in  HBase
       Assert.assertEquals(1, returnedCells.size());
       // Assert on cell contents
       Assert.assertEquals(0, Bytes.compareTo(
           CellUtil.cloneFamily(returnedCells.get(0)), CF1));
       Assert.assertEquals(0, Bytes.compareTo(CellUtil.cloneQualifier(returnedCells.get(0)), COL_QUALIFIER));
-      Assert.assertEquals(0, Bytes.compareTo(CellUtil.cloneValue(returnedCells.get(0)), (VALUE_PREFIX + i).getBytes()));
-      i++;
+      Assert.assertEquals(0, Bytes.compareTo(CellUtil.cloneValue(returnedCells.get(0)), (VALUE_PREFIX + rowStart).getBytes()));
+      rowStart++;
```

Comment:
let rowStart be the start. Take an iterator int and increment it. It looks cleaner and more readable.